src/components/Alerts/alerts.util.js

```diff
@@ -71,46 +80,142 @@ export const parseAlertsQueryParamsCallback = (paramName, paramValue) => {
   }
 
   if (paramName === EVENT_TYPE) {
-    return filterAlertsEventTypeOptions.find(type => type.id === paramValue)?.id
+    return alertsEventTypeOptions.find(type => type.id === paramValue)?.id
   }
 
   return paramValue
 }
 
+export const generatePageData = (handleFetchJobLogs, selectedAlert) => {
+  return {
+    page: ALERTS_PAGE,
+    details: {
+      type: ALERTS_PAGE,
+      entityType: selectedAlert.entity_kind,
+      infoHeaders: alertsHeaders(selectedAlert.entity_kind),
+      menu: [],
+      notifications,
+      refreshLogs: handleFetchJobLogs,
+      triggerCriteria: triggerCriteria(selectedAlert.criteria)
+    },
+    selectedAlert
```

Comment:
why do you pass selected item here?

Reply:
it used in generateAlertsDetailsInfo function.
i modify the code :
`triggerCriteria: triggerCriteria
`
....

```
    const triggerCriteriaContent = pageData?.details?.triggerCriteria(
      pageData?.selectedAlert?.criteria
    )

```

Reply:
Moved triggerCriteria logic to detailsInfo for better organization.
Replaced all instances of pageData with selectedItem.
Let me know if this includes all the changes or if more adjustments are needed.